langchain/cache.py

```diff
@@ -163,7 +179,7 @@ def update(self, prompt: str, llm_string: str, return_val: RETURN_VAL_TYPE) -> N
     def clear(self, **kwargs: Any) -> None:
         """Clear cache."""
         with Session(self.engine) as session:
-            session.execute(self.cache_schema.delete())
+            session.query(self.cache_schema).delete()
```

Comment:
Why is this changing ?

Reply:
yeah not sure, from previous pr, ill revert